apps/test-pololu370-encoder/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(test-pololu370-encoder.${TARGET}-board.elf)
+target_sources(test-pololu370-encoder.${TARGET}-board.elf PRIVATE src/main.cpp)
+target_link_libraries(test-pololu370-encoder.${TARGET}-board.elf PRIVATE EncoderPololu37D QEI Logger)
+target_set_firmware_properties(test-pololu370-encoder.${TARGET}-board.elf)
```

Comment:
Typo here, it should be `pololu37D` instead of `pololu370`, make sure to replace all instances of these

Reply:
ohh i just noticed that, Its giving me an error when i try to rename it. I included the error in the discord channel.